frontend/src/queries/query.ts

```diff
@@ -382,13 +382,14 @@ export async function legacyInsightQuery({
 }: LegacyInsightQueryParams): Promise<[Response, string]> {
     const apiUrl = legacyInsightQueryURL({ filters, currentTeamId, refresh })
```

Comment:
Hmm.. the url is generated here and still contains all filters in query params. So this would still overflow the url length limit (while sending the filters in the request body as well) if I'm not mistaken.